connectors/class-connector-media.php

```diff
@@ -151,7 +151,7 @@ public function callback_add_attachment( $post_id ) {
 		$url             = $post->guid;
 		$parent_id       = $post->post_parent;
 		$parent          = get_post( $parent_id );
-		$parent_title    = $parent_id ? $parent->post_title : null;
+		$parent_title    = $parent instanceof \WP_Post ? $parent->post_title : 'Unidentifiable post';
```

Comment:
@kasparsd Please take a look at this :point_up:. It resolves #907. I also updated the corresponding test to test this logic as well.